bitnami/thanos/values.yaml

```diff
@@ -1778,6 +1778,16 @@ storegateway:
     ##
     # maxUnavailable: 1
 
+  ## Setting hashPartioning will create multiple store deployments based on the number of shards specified using the hashmod of the blocks
+  ## See https://thanos.io/tip/thanos/sharding.md/#relabelling
+  hashPartioning: {}
```

Comment:
Although this works with this:
```
{{- if .Values.storegateway.hashPartioning }}
  {{- $shards = int .Values.storegateway.hashPartioning.shards }}
  {{- $hashPartioning = true }}
{{- else }}
  {{- $shards = len .Values.storegateway.timePartioning }}
  {{- $timePartioning = true }}
{{- end }}
```
I'd the `values.yaml` should be descriptive and include the allowed values.
Could you please modify the logic in the statefulset so the `values.yaml` can be set similar to this?
```
  hashPartioning:
    shards: ""
```

Reply:
Done